server/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:1.14.2-alpine3.11 AS build
+# Needed to install go-sqlite3
+RUN apk add --update gcc musl-dev
+COPY ./server/ /go/src/beavertail/server
+COPY ./protocols/ /go/src/beavertail/protocols
+WORKDIR /go/src/beavertail/server
+RUN go get -v -d .
+RUN go build -v
+
+FROM alpine:3.11.5
+COPY --from=build /go/src/beavertail/server/server ./server
+RUN mkdir /db/
+EXPOSE 3000
+ENTRYPOINT ["./server"]
+CMD ["-db", "/db/taps.db"]
```

```suggestion
CMD ["-db", "/db/taps.db", "-host", "0.0.0.0", "-port", "-3000"]
```

Since we're exposing port 3000 we should listen on that port, the `server` binary listens on port 8080 by default. 
We'll also want to make sure our container is accessible from the outside world so we'll have to bind to `0.0.0.0` ([Ref](https://stackoverflow.com/questions/47025647/localhost-vs-0-0-0-0-with-docker-on-mac-os))

It might be useful to have a config.yml, but I think the flags are a good way of doing it for right now

---

Yikes, I missed that! Thanks. (That was the whole reason I asked to make it configurable, too...)